frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardParentToMatchBranchAction.java

```diff
@@ -71,7 +71,7 @@ public void update(AnActionEvent anActionEvent) {
     if (syncToParentStatus == SyncToParentStatus.InSync) {
 
       if (getCurrentBranchNameIfManaged(anActionEvent).equals(branchName)) {
-        presentation.setText("Fast Forward Parent To Match Current Branch");
+        presentation.setText("_Fast Forward Parent To Match Current Branch");
```

Comment:
Btw those "dynamic" descriptions could be extracted to the bundle as well... I'm gonna open an issue for that

Reply:
Good point! I thought about it once but sadly forgot XD We could bundle not only texts and descs but vcs notifications too! :rocket: